static-site/src/components/ListResources/index.tsx

```diff
@@ -73,6 +72,12 @@ function ListResources({
     )
   }
 
+  const tiles = pathogens.map((p) => ({
+    name: p.name,
+    img: p.img,
+    filters: [p.id]
+  } as FilterTile));
```

Comment:
why is this cast needed?

Reply:
I've fixed my intentions in [force-push](https://github.com/nextstrain/nextstrain.org/compare/f9a617103fb9f2a22ab0277f94fd1b3649e646df..5c46de7e9269ca8a64bbf3863de9e8b196a91f0a). Hopefully this makes more sense: any issues with the map output (e.g. missing `filters`) will show a type error here instead of further down.